### ISSUE TITLE
SPARC Build

### ISSUE THREAD

**Issue by Asbestosstar** [psumbera/solaris-openjdk#1]
Hi, I saw in your git history that you made some changes for a SPARC build, do you plan to support SPARC in the future?
I know tribbelx does not, do you plan to readd the sparc stuff?

**Comment by psumbera** [comment]
No. I have no plans for SPARC. It would be huge effort for which I have no time.
